<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpanCalibration</class>
 <widget class="QWidget" name="SpanCalibration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QwtPlot" name="CalibrationPlot">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>541</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SpanCalButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>400</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibrate Span</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PotCalButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>440</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibrate Potentiometer</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="Instructions">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>30</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="curSpanDisplay">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>400</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="setTotSpanButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>430</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Cal Point</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="setTotSpan">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>400</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>380</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Target Inches</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>380</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current total span</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="activatePotCal">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>460</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Activate Pot Cal</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
